app/assets/javascripts/base.js.erb

```diff
@@ -4,6 +4,7 @@
 //= require initializePage
 //= require utilities/getImageForLink
 //= require @honeybadger-io/js/dist/browser/honeybadger.js
+//= require @hotwired/turbo
```

Comment:
This is causing the specs to fail. If I remove this locally and run a spec that's failing on Travis it will pass. If I put it back, it fails. So confused. 

Spec I'm running: `bundle exec rspec spec/system/user/user_self_destroy_spec.rb`

failure:
```
 1) User destroys their profile destroys an account
     Failure/Error: expect(matching_job).to be_present, "No enqueued job found with #{expected}"
       No enqueued job found with {:job=>Users::DeleteWorker}
```